internal/constants/constants.go

```diff
@@ -109,4 +111,7 @@ const (
 	PathAppend             = "/usr/bin:/usr/sbin:/bin:/sbin"
 	PATH                   = "PATH"
 	DefaultPCR             = 11
+	SourceSysExtDir        = "/.extra/sysext/"
+	DestSysExtDir          = "/var/lib/extensions"
```

Comment:
I think we may not have to do this. We can create and copy them under /run/extensions so they are clear on each reboot instead of keeping them in persistent. Hence we have just one place where we copy them to update and such, the EFI partition. Then when a new version is released you can drop it there and remove the old one